src/TextBox/text-box-style.scss

```diff
@@ -1,4 +1,10 @@
-$red: #F03529;
+$red: #f03529;
+$grey6: #999999;
+$blue: #41b0ee;
```

Comment:
maybe you can use kebab case syntax similar to SEB bootstrap repo:
`$blue:                  #41B0EE;`
`$blue-dark:             #0092E1; // SEB specific`
`$blue-darker:           #007AC7; // SEB specific`

Reply:
resolved in commit https://github.com/sebgroup/react-components/pull/29/commits/2b668852b23d63790e2f37e7a82001a8673b8ad5